### DISCUSSION TITLE
Copyright symbol in Markdown

### DISCUSSION THREAD

**Discussion by franciskinyuru** [community/community#34676] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi I am trying to add the copyright symbol in my READme.md file and have not been successful.
Its HTML represenation is  &copy;.

What is the markdown equivalent.

**Comment by SilasRodrigues19** [comment, marked as answer]
@franciskinyuru use ```&copy;```

output: 
&copy;

**Reply by franciskinyuru** [reply]
Works Okay Thanks @SilasRodrigues19

**Reply by SilasRodrigues19** [reply]
@franciskinyuru No worries.

**Reply by createme506** [reply]
If I want to buy some Api's like MT4 and MT6 where do I go
